product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -70,11 +70,11 @@ performance.
 
```

Comment:
Above, line 29, this sentence: You can only change
[`bdr.max_writers_per_subscription`](/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription)
with a server restart.

Can this be rewritten in the positive? Would this be accurate? 

Changing [`bdr.max_writers_per_subscription`](/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription) requires a server restart.

Reply:
Changing [bdr.max_writers_per_subscription](https://github.com/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription) requires a server restart to take effect.